app/src/main/java/com/puzzlebench/clean_marvel_kotlin/presentation/mvp/CharacterModel.kt

```diff
@@ -0,0 +1,77 @@
+package com.puzzlebench.clean_marvel_kotlin.presentation.mvp
+
+import android.util.Log
+import com.puzzlebench.clean_marvel_kotlin.CONSTS
+import com.puzzlebench.clean_marvel_kotlin.domain.model.Character
+import com.puzzlebench.clean_marvel_kotlin.domain.model.RealmCharacter
+import com.puzzlebench.clean_marvel_kotlin.domain.model.RealmThumbnail
+import com.puzzlebench.clean_marvel_kotlin.domain.model.Thumbnail
+import io.realm.Realm
+import io.realm.kotlin.createObject
+import io.realm.kotlin.where
+
+class CharacterModel {
+
+    lateinit var realm: Realm
+
+    fun init() {
+        realm = Realm.getDefaultInstance()
+    }
+
+    fun getCharacters(): List<Character> {
+        if (realm.where<RealmCharacter>().findAll().size > 0) {
+            return realmGetter()
+        }
+        return mutableListOf()
+    }
+
+    private fun realmGetter(): List<Character> {
```

Comment:
this is a problem of the UseCase.. 
the model call the useCase, and the useCase call the DB

Reply:
Done